tarodb/assets_store.go

```diff
@@ -238,7 +238,7 @@ type AssetFamilyBalance struct {
 type BatchedAssetStore interface {
 	ActiveAssetsStore
 
-	BatchedTx[ActiveAssetsStore, TxOptions]
```

Comment:
I tried this and failed to get it working with Generics... But I changed the signature to use `sqlc.DBTX` instead of the `*sql.Tx` reference directly. Or was that what you had in mind?